lib/Compilation.js

```diff
@@ -1472,7 +1472,9 @@ BREAKING CHANGE: Asset processing hooks in Compilation has been merged into a si
 
 			for (const item of sortedDependencies) {
 				inProgressTransitive++;
+				this.buildQueue.increaseParallelism();
```

Comment:
This is not correct here, as we are not blocking the buildQueue during `_processModuleDependencies`.